src/scripts.js

```diff
@@ -502,4 +503,11 @@ function cookRecipe() {
    removeIngredientsFromPantry().forEach(element => {
         updatePantry(element)
     })
+}
+
+function fixEnter(event){
+    if(event.keyCode === 13){
+        event.preventDefault();
+        addItemToPantry()
+    }
```

Comment:
Do you think that we should just put this function in the event listener like the searchBar one? Something like this? 
inputQuantity.addEventListener('keydown', (event) => {
  if(event.keyCode === 13) {
        event.preventDefault();
        addItemToPantry()
    }
})

Reply:
Ty for this feedback, it is now an implicit function